pkgs/applications/video/ccextractor/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub , pkgconfig, cmake
+, glew, glfw3, leptonica, tesseract3, zlib }:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  name = "ccextractor-${version}";
```

Comment:
Please set to `pname = "ccextractor";` and then `name` will be taken care of for you.